src/js/themes/base.d.ts

```diff
@@ -820,6 +820,9 @@ export interface ThemeType {
   grommet?: {
     extend?: ExtendType;
   };
+  header?: {
+    zIndex?: string;
```

Comment:
I think this should be `header?: { sticky?: { zIndex?: string; } }`

Reply:
aw yes you are correct

Reply:
done